pkg/scheduler/actions/elect/elect.go

```diff
@@ -0,0 +1,52 @@
+// Package elect is used to find the target job and reserve resource for it
+package elect
+
+import (
+	"k8s.io/klog"
+	"volcano.sh/volcano/pkg/apis/scheduling"
+	"volcano.sh/volcano/pkg/scheduler/api"
+	"volcano.sh/volcano/pkg/scheduler/framework"
+	"volcano.sh/volcano/pkg/scheduler/util"
+)
+
+// Action defines the action
+type Action struct{}
+
+// New returns the action instance
+func New() *Action {
+	return &Action{}
+}
+
+// Name returns the action name
+func (alloc *Action) Name() string {
+	return "elect"
+}
+
+// Initialize inits the action
+func (alloc *Action) Initialize() {}
+
+// Execute selects the target job which is of the highest priority and waits for the longest time.
+func (alloc *Action) Execute(ssn *framework.Session) {
+	klog.V(3).Infof("Enter Elect ...")
+	defer klog.V(3).Infof("Leaving Elect ...")
+
+	if util.Reservation.TargetJob == nil {
+		klog.V(3).Infof("Start select Target Job...")
+		var pendingJobs []*api.JobInfo
+		for _, job := range ssn.Jobs {
+			klog.V(3).Infof("job: %s, pgStatus: %s", job.Name, job.PodGroup.Status.Phase)
```

Comment:
suggest use log `V(4)` to avoid scheduler log  grows too quickly.  Please also check if log level is appropriate in other code block.

Reply:
> suggest use log `V(4)` to avoid scheduler log grows too quickly. Please also check if log level is appropriate in other code block.

OK，I'll check all logs and remove noisy log later.